### DISCUSSION TITLE
Display issue with WooCommerce media library

### DISCUSSION THREAD

**Discussion by Obustin** [woocommerce/woocommerce#39370] [WooCommerce New Product Editor]
Hello,

As you can see, I have a display issue when trying to add a product. The media-sidebar is not displaying correctly. I have tried disabling all plugins except for WooCommerce, changing themes, and PHP versions, but the problem persists. 

Does anyone have any suggestions to resolve this issue? Thank you very much.

<img width="2121" alt="Woocommerce Media" src="https://github.com/woocommerce/woocommerce/assets/135419954/7c356e41-8159-4210-bf85-1df18087e059">
<img width="846" alt="Capture d’écran 2023-07-22 à 10 46 30" src="https://github.com/woocommerce/woocommerce/assets/135419954/016105c6-f9bf-40c8-8a95-cafda9f2d3ed">

**Comment by louwie17** [comment]
Hi @Obustin, is this issue specifically happening with the new product block editor? It appears the above media sidebar is only accessible through the current WP product editor.
It may be better to create an issue for this [here](https://github.com/woocommerce/woocommerce/issues/new/choose), and adding some instructions on how to reproduce this.
